dot/rpc/modules/author_test.go

```diff
@@ -353,9 +353,9 @@ func TestAuthorModule_PendingExtrinsics(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
-		res := PendingExtrinsicsResponse{}
-		tt.args.res = &res
 		t.Run(tt.name, func(t *testing.T) {
+			res := PendingExtrinsicsResponse{}
+			tt.args.res = &res
```

Comment:
Why not set `res: &PendingExtrinsicsResponse{}` directly in the `tests` declaration above by the way? :thinking: 

Same comment for other tests

Reply:
I think this would work, will change

Reply:
Actually on second thought, this will add a lot of lines so I'm not sure that its worth doing this change. If you think it is I will though

Reply:
See https://github.com/ChainSafe/gossamer/pull/1995#discussion_r756771003